bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/EmotivaUdpSendingService.java

```diff
@@ -196,14 +196,14 @@ public void send(String msg) throws IOException {
                 logger.debug("Sending successful");
 
                 localDatagramSocket.receive(answer);
-                final byte[] receivedData = Arrays.copyOfRange(answer.getData(), 0, answer.getLength() - 1);
+                final int receivedData = Arrays.copyOfRange(answer.getData(), 0, answer.getLength() - 1).length;
```

Comment:
```suggestion
                final int receivedDataLength = Arrays.copyOfRange(answer.getData(), 0, answer.getLength() - 1).length;
```

Minor, but better to be consistent.